village/src/components/SmurfForm.js

```diff
@@ -9,16 +10,17 @@ class SmurfForm extends Component {
       height: ''
     };
   }
-
+  // add code to create the smurf using the api
   addSmurf = event => {
     event.preventDefault();
```

Comment:
You'll want to put this on App.js so that you can update the state that should be used by the entire application. As of right now, the data must reload every time the route reloads.